bin/build

```diff
@@ -28,5 +28,5 @@ echo 'finished'
 # build API documentation
 echo 'building api documentation...'
 cd ./src/api-docs
-bundle install
-bundle exec middleman build --clean --build-dir=../../build/api-docs
\ No newline at end of file
+bundle install --path=../../build/vendor
```

Comment:
hm.. This is in the folder that is being served for HTTP, maybe I should move this into somewhere else and gitignore it?